modules/auxiliary/admin/networking/mikrotik_config.rb

```diff
@@ -0,0 +1,43 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/auxiliary/mikrotik'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Mikrotik
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Mikrotik Configuration Importer',
+        'Description' => %q{
+          This module imports a Mikrotik device configuration.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'h00die']
+      )
+    )
+
+    register_options(
+      [
+        OptPath.new('CONFIG', [true, 'Path to configuration to import']),
+        Opt::RHOST(),
+        Opt::RPORT(22)
+      ]
+    )
+
+  end
+
+  def run
+    unless ::File.exist?(datastore['CONFIG'])
+      fail_with Failure::BadConfig, "Mikrotik config file #{datastore['CONFIG']} does not exists!"
```

Comment:
Unofficial and disputed style requirements mandate the use of parenthesis - which is in line with the style used for the remainder of this module.

```suggestion
      fail_with(Failure::BadConfig, "Mikrotik config file #{datastore['CONFIG']} does not exists!")
```